src/widgets/refinement-list/refinement-list.js

```diff
@@ -247,6 +249,9 @@ export default function refinementList(
     throw new Error('showMore.limit configuration should be > than the limit in the main configuration'); // eslint-disable-line
   }
 
+  const escapeFacetValues = searchForFacetValues
+    ? searchForFacetValues.escapeFacetValues || false
```

Comment:
Maybe cast the value with `Boolean()`, it will ensure that the value will be a `boolean`. With the actual condition the value can be anything thruthy or a `boolean`.